tests/components/test_power_output.py

```diff
@@ -0,0 +1,80 @@
+"""Tests for the power output classes."""
+from typing import List, Type
+
+from j5.backends import Backend
+from j5.boards import Board
+from j5.components.power_output import PowerOutput, PowerOutputInterface
+
+
+class MockPowerOutputDriver(PowerOutputInterface):
+    """A testing driver for power outputs."""
+
+    def __init__(self):
+        self._enabled = False
+
+    def get_power_output_enabled(self, board: Board, identifier: int) -> bool:
+        """Get whether a power output is enabled."""
+        return self._enabled
+
+    def set_power_output_enabled(
+        self, board: Board, identifier: int, enabled: bool,
+    ) -> None:
+        """Set whether a power output is enabled."""
+        self._enabled = enabled
+
+    def get_power_output_current(self, board: Board, identifier: int) -> float:
+        """Get the current being drawn on a power output, in amperes."""
+        return 8.1
+
+
+class MockPowerOutputBoard(Board):
+    """A testing board for the power output."""
+
+    @property
+    def name(self) -> str:
+        """The name of this board."""
+        return "Testing Power Output Board"
+
+    @property
+    def serial(self) -> str:
+        """The serial number of this board."""
+        return "SERIAL"
+
+    @property
+    def supported_components(self) -> List[Type["Component"]]:
```

Comment:
`Component` doesn't have to be in quotes, though I'm happy for them to be left if you'd rather not add an additional import to the file

Reply:
I copy-pasted this from another test file, so I think I'll leave it as is. We can change them all later if we think that's the right thing to do.